frontEnd/src/components/EmojiButton/EmojiButton.module.css

```diff
@@ -0,0 +1,14 @@
+.EmojiButton {
+  height: 100px;
+  width: 100px;
+  margin: 20px;
+  background-color: lightgray;
```

Comment:
Could optionally collect the colours in css variables in a `:root` block somewhere for easy reuse. Again, very minor.